### ISSUE TITLE
Documentation culture settings are ignored in the stylecop.json in JetBrains Rider

### ISSUE THREAD

**Issue by artem-voropaev** [DotNetAnalyzers/StyleCopAnalyzers#2958]
I use next settings in the stylecop.json with StyleCop Analyzers 1.1.118

```json  
{
    "documentationRules": {
        "documentationCulture": "ru-RU"
    }
}
```
StyleCop warns me [SA1623] The property's documentation summary text should begin with: 'Gets or sets', but I use "documentationCulture": "ru-RU" and it should not warn me because my code is correct for this culture
```csharp    
/// <summary>
/// Получает или задает что-то.
/// </summary>
public string Foo { get; set; }
```

"Gets or sets" is "Получает или задает" in ru-RU (it always worked in Visual Studio with Settings.StyleCop file for .Net Framework projects)

My csproj
```xml
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <TargetFramework>netstandard2.0</TargetFramework>
        <CodeAnalysisRuleSet>../main.ruleset</CodeAnalysisRuleSet>
        <DocumentationFile>$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
    </PropertyGroup>

    <ItemGroup>
      <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" />
    </ItemGroup>

    <ItemGroup>
        <AdditionalFiles Include="../stylecop.json" />
    </ItemGroup>
    
</Project>

````

**Comment by sharwell** [comment]
Is this warning also reported when building from the command line?

**Comment by sharwell** [comment]
I just verified that this is working in Visual Studio with StyleCop Analyzers 1.1.118. Closing as **external**; this bug should be reported to JetBrains.

**Comment by artem-voropaev** [comment]
> Is this warning also reported when building from the command line?

Yes, dotnet build TestAnalyzers.sln -v n
warning SA1623: The property's documentation summary text should begin with: 'Gets or sets'

It seems that is Stylecop.Analyzers bug

**Comment by sharwell** [comment]
@artem-voropaev What version of the dotnet tool are you using? It sounds like https://github.com/dotnet/roslyn/issues/29014.

**Comment by artem-voropaev** [comment]
> @artem-voropaev What version of the dotnet tool are you using? It sounds like [dotnet/roslyn#29014](https://github.com/dotnet/roslyn/issues/29014).

dotnet --info

Пакет SDK для .NET Core (отражающий любой global.json):
 Version:   2.1.503
 Commit:    4c506e0f35

Среда выполнения:
 OS Name:     Mac OS X
 OS Version:  10.14
 OS Platform: Darwin
 RID:         osx.10.14-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.1.503/

Host (useful for support):
  Version: 2.1.7
  Commit:  cca5d72d48

.NET Core SDKs installed:
  2.1.4 [/usr/local/share/dotnet/sdk]
  2.1.302 [/usr/local/share/dotnet/sdk]
  2.1.503 [/usr/local/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.2 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.7 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.2 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.7 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.5 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.2 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.7 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

**Comment by artem-voropaev** [comment]
Why you close this issue? The problem was not solved! I'm waiting for solution
